test/queries/developer_query_test.rb

```diff
@@ -181,6 +181,7 @@ class DeveloperQueryTest < ActiveSupport::TestCase
     assert_includes records, stimulus_developer
     refute_includes records, react_developer
     refute_includes records, developer
+    assert_equal records.length, records.uniq.length
```

Comment:
Ensure there is no record duplication. Prior to the change above, the addition of this test was causing a failure in the test suite because the `hotwire_developer` was selected twice since they had more than 1 matching specialty.